app/serializers/error_serializer.rb

```diff
@@ -20,4 +20,15 @@ def login_error
         }
     }
   end
+
+  def unauthorized_error
+    {
+      data:
+        {
+          id: nil,
+          type:"error",
+          error_message: 'Credentials are bad'
+        }
+    }
```

Comment:
I am planning to refactor this class because I am repeating myself in each one of these methods